### DISCUSSION TITLE
I made a static website generator to render ZAP files

### DISCUSSION THREAD

**Discussion by zakaria1193** [project-chip/zap#1239] [General topics]
Our company makes  many many Zigbee devices, and we're always requested list of clusters and attributes etc available on our devices. We used to stored that as manually maintained documentation. But since we started using ZAP, why write the documentation manually? specially since descriptions can be fetched from ZCL xml files to make every doc more readable !

Check it out at
https://github.com/zakaria1193/zap-gatsby

Live view on github pages of master https://zakaria1193.github.io/zap-gatsby/

Everything is open for discussion. I'm originally a firmware engineer so my web skills are noob level.

Matter support is not ready since i needed it mostly for Zigbee devices. but contributions are welcome !

**Comment by paulr34** [comment]
@zakaria1193 this is cool!
